### PR TITLE
feat(ttl): Clearly identify ephemeral server groups

### PR DESCRIPTION
**Pull request by ajordens**
image

### CODE REVIEW

**Comment by ajordens** [comment]
No console errors when looking at an application containing server groups w/ and w/o `spinnaker:ttl` entity tags.

@anotherchrisberry How do I go about a colored header in the `HoverablePopover`? The clock icon in the side bar is slightly grayed out ... 

I opt'd for the `HoverablePopover` since at some point we'll add a delete option.

File: app/scripts/modules/core/src/entityTag/notifications/EphemeralPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    this.state = this.getState(props);
+  }
+
+  private getState(props: IEphemeralPopoverProps): IEphemeralPopoverState {
+    const entityTags: IEntityTags = props.entity.entityTags;
+    if (entityTags) {
+      const ephemeralTag = entityTags.tags.filter(x => x.name === 'spinnaker:ttl')[0];
+      if (ephemeralTag) {
+        return {
+          destroyedFromNow: moment(ephemeralTag.value.expiry).fromNow(true),
```
Comment (ajordens) [inline app/scripts/modules/core/src/entityTag/notifications/EphemeralPopover.tsx]: pretty trusting that `expiry` is a legitimate value (`"expiry": "2018-05-15T22:34:42.422Z"`)

**Comment by ajordens** [comment]
Actually I think I can control the header if I pass in an actual `Component` to `HoverablePopover`.

**Comment by ajordens** [comment]
image

Revised color scheme -- went one shade darker than what we were using for the account label. No strong opinions on color or title text (`Ephemeral Server Group` ??)

File: app/scripts/modules/core/src/entityTag/notifications/notifications.less [hunk 2]
```diff
@@ -1,6 +1,8 @@
-@import (reference) "~core/presentation/less/imports/commonImports.less";
+@import (reference) '~core/presentation/less/imports/commonImports.less';
```
Comment (ajordens) [inline app/scripts/modules/core/src/entityTag/notifications/notifications.less]: I think the fancy post-commit hook fixed this up?

Comment (anotherchrisberry) [reply app/scripts/modules/core/src/entityTag/notifications/notifications.less]: Yeah, that is prettier doing its thing.

File: app/scripts/modules/core/src/entityTag/notifications/EntityNotifications.tsx [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { Placement } from 'core/presentation';
 import { Application } from 'core/application';
 import { noop } from 'core/utils';
 import { NotificationsPopover } from './NotificationsPopover';
+import { EphemeralPopover } from 'core/entityTag/notifications/EphemeralPopover';
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/entityTag/notifications/EntityNotifications.tsx]: can shorten this to:
```javascript
import { EphemeralPopover } from './EphemeralPopover';
```

**Comment by anotherchrisberry** [comment]
How would you feel about https://fontawesome.com/icons/bomb?style=solid as the icon? Too violent?
